fs/march/multierror.go

```diff
@@ -0,0 +1,40 @@
+package march
+
+import (
+	"bytes"
+	"fmt"
+	"sync"
+)
+
+type multierror struct {
```

Comment:
Why do we need a new multierror type? 

How about using an existing package like one from Uber or HashiCorp?

Reply:
We need protection for concurrent use I don't think those packages provide that?

Reply:
I think I'd be happy just showing the last error (or the first).  More than a couple of errors are going to make the error messages unreadable.